docsets/RxJava/docset.json

```diff
@@ -1,11 +1,24 @@
 {
     "name": "RxJava",
-    "version": "1.1.6",
+    "version": "2.0.6",
     "archive": "RxJava.tgz",
     "author": {
         "name": "XiNGRZ",
         "link": "https://github.com/xingrz"
     },
     "aliases": [],
-    "specific_versions": []
+    "specific_versions": [
+      {
+        "version": "2.0.6",
+        "archive": "versions/2.0.6/RxJava.tgz"
+      },
+      {
+        "version": "1.2.7",
+        "archive": "versions/1.2.7/RxJava.tgz"
+      },
+      {
+        "version": "1.1.6",
+        "archive": "versions/1.1.6/RxJava.tgz"
```

Comment:
You need to add a docset archive at this path too. Moving the `txt` file doesn't work.